src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtworkLocationFilter.tsx

```diff
@@ -36,14 +36,19 @@ const ArtworkLocationOption: React.FC<{ name: string }> = ({ name }) => {
 }
 
 export const ArtworkLocationFilter: FC = () => {
-  const { aggregations } = useArtworkFilterContext()
+  const { aggregations, currentlySelectedFilters } = useArtworkFilterContext()
   const locations = aggregations.find(agg => agg.slice === "LOCATION_CITY")
 
   if (!(locations && locations.counts)) {
     return null
   }
 
   const locationsSorted = sortBy(locations.counts, ["count"]).reverse()
+  const hasLocationFilter =
+    intersection(
+      currentlySelectedFilters().locationCities,
+      locations.counts.slice(INITIAL_ITEMS_TO_SHOW).map(({ name }) => name)
+    ).length > 0
```

Comment:
Oh, I had to read this a couple of times before I realized what was happening here. 

For readability I'd like to suggest pulling these intermediate values into named variables?

```js
const hasLocationFilter =
   intersection(
     currentlySelectedFilters().locationCities,
     locations.counts.slice(INITIAL_ITEMS_TO_SHOW).map(({ name }) => name)
   ).length > 0
```

-vs-

```js
const selectedOptions = currentlySelectedFilters().locationCities
const belowTheFoldOptions = locations.counts.slice(INITIAL_ITEMS_TO_SHOW).map(({ name }) => name)

const hasBelowTheFoldSelections =
   intersection(selectedOptions, belowTheFoldOptions).length > 0
```

Reply:
yea, I think `hasBelowTheFold...` naming might make better sense. `hasLocationFilter` is misleading, that will be `false` if you've selected an above the fold filter.